arangod/Cluster/Maintenance.cpp

```diff
@@ -466,8 +466,10 @@ static void handleLocalShard(
     if (replicationVersion != replication::Version::TWO) {
```

Comment:
Activating this PR would mean an `|| isLeading` inside this conditional, as far as I see.

Reply:
😵‍💫 why is it not in this PR but was in a different one...